### PR TITLE
Update sauce-connect-proxy.md

### PR DESCRIPTION
**Pull request by mfranzonello2**
added clarification around filepath format for Mac/Linux vs Windows for --pac flag

### Description
Added examples with 2-vs-3 slashes for filepath in Mac/Linux vs Windows environments and removed Windows-only callout

### Motivation and Context
Customer confusion around why their SC tests were failing, and inconsistent Sauce documentation.

### Types of Changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Documentation fix (typos, incorrect content, missing content, etc.)

### CODE REVIEW

File: docs/dev/cli/sauce-connect-proxy.md [hunk 1]
```diff
@@ -243,17 +243,34 @@ You can specify a combination of several options. For example:
 
 <p><small>| OPTIONAL | STRING | <span className="sauceGreen">4.7.x</span> <span className="sauceGreen">4.8.x</span> <span className="sauceGreen">4.9.x</span> | </small></p>
 
-**Description**: Defines external proxy auto-configuration (PAC) URI. You can input `http(s)` or `local file://URL`. Absolute paths are required when specifying a local PAC file. `--pac` supports the standard file URL format (that is, file://auth-path/local-path). On Windows, the format looks like `file:///C:/path/to/file`. For more information, see [Sauce Connect Proxy Setup with Additional Proxies](/secure-connections/sauce-connect/setup-configuration/additional-proxies).<br/>
+**Description**: Defines external proxy auto-configuration (PAC) URI. You can input `http(s)` or `local file://URL`. Absolute paths are required when specifying a local PAC file. `--pac` supports the standard file URL format (that is, file://auth-path/local-path, see below for differences between Mac/Linux and Windows). For more information, see [Sauce Connect Proxy Setup with Additional Proxies](/secure-connections/sauce-connect/setup-configuration/additional-proxies).<br/>
```
Comment (LoryApiFortress) [inline docs/dev/cli/sauce-connect-proxy.md:246]:
```suggestion
**Description**: Defines external proxy auto-configuration (PAC) URI. You can input `http(s)` or `local file://URL`. Absolute paths are required when specifying a local PAC file. `--pac` supports the standard file URL format (that is, file://auth-path/local-path). On Windows, the format looks like `file:///C:/path/to/file`. For more information, see [Sauce Connect Proxy Setup with Additional Proxies](/secure-connections/sauce-connect/setup-configuration/additional-proxies).<br/>
```

**Review by LoryApiFortress — APPROVED** [review]

File: docs/dev/cli/sauce-connect-proxy.md [hunk 2]
```diff
@@ -243,17 +243,34 @@ You can specify a combination of several options. For example:
...
+```bash
 --pac file://Users/JohnSmith/Desktop/MyPac.pac
 ```
+
+</TabItem>
+
+<TabItem value="windows">
+
+```bash
+--pac file:///Users/JohnSmith/Desktop/MyPac.pac
```
Comment (amckenzie132) [inline docs/dev/cli/sauce-connect-proxy.md:268]: `--pac file:///C:/Users/JohnSmith/Desktop/MyPac.pac`

* Windows needs the drive

File: docs/dev/cli/sauce-connect-proxy.md [hunk 3]
```diff
@@ -247,13 +247,30 @@ You can specify a combination of several options. For example:
...
```
Comment (LoryApiFortress) [inline docs/dev/cli/sauce-connect-proxy.md:268]:
```suggestion
--pac file:///C:/Users/JohnSmith/Desktop/MyPac.pac
```

File: docs/dev/cli/sauce-connect-proxy.md (hunk 2 above)
Comment (LoryApiFortress) [reply docs/dev/cli/sauce-connect-proxy.md]: Thank you Alana!

File: docs/dev/cli/sauce-connect-proxy.md [hunk 4]
```diff
@@ -243,17 +243,34 @@ You can specify a combination of several options. For example:
...
+values={[
+{label: 'Mac or Linux', value: 'maclinux'},
+{label: 'Windows', value: 'windows'},
+]}>
+
+<TabItem value="maclinux">
 
-```bash On Linux/Mac
+```bash
 --pac file://Users/JohnSmith/Desktop/MyPac.pac
```
Comment (amckenzie132) [inline docs/dev/cli/sauce-connect-proxy.md:260]: `--pac file:///Users/JohnSmith/Desktop/MyPac.pac`

* for local files, `auth-path` is empty

**Pull request merged**